transaction-context/src/transaction_accounts.rs

```diff
@@ -2,26 +2,249 @@
 use qualifier_attr::qualifiers;
 use {
     crate::{IndexOfAccount, MAX_ACCOUNT_DATA_GROWTH_PER_TRANSACTION, MAX_ACCOUNT_DATA_LEN},
-    solana_account::AccountSharedData,
+    solana_account::{AccountSharedData, ReadableAccount, WritableAccount},
     solana_instruction::error::InstructionError,
     solana_pubkey::Pubkey,
     std::{
         cell::{Cell, UnsafeCell},
         ops::{Deref, DerefMut},
+        ptr,
+        sync::Arc,
     },
 };
 
+#[repr(C)]
+#[derive(Debug, PartialEq)]
+struct SVMAccount {
+    key: Pubkey,
+    owner: Pubkey,
+    lamports: u64,
+    // The payload is going to be filled with the guest virtual address of the account payload
+    // vector.
+    // payload: VmSlice,
```

Comment:
This is commented out, since we are not yet using this field.

Reply:
Populated in https://github.com/anza-xyz/agave/pull/8372/commits/33b9698518d22bd4373f55bdd1ba1a1f65058947